crates/astria-sequencer-relayer/local.env.example

```diff
@@ -45,6 +45,18 @@ ASTRIA_SEQUENCER_RELAYER_RELAY_ONLY_VALIDATOR_KEY_BLOCKS=false
 # Ignored if `ASTRIA_SEQUENCER_RELAYER_DISABLE_RELAY_ALL=false`.
 ASTRIA_SEQUENCER_RELAYER_VALIDATOR_KEY_FILE=.cometbft/config/priv_validator_key.json
 
+# The path to which relayer will write its state prior to submitting to Celestia.
+# A file must exist at this path and the file must be readable and writable.
+# The file must either be empty or contain a JSON object that can be read by
+# by Relayer (this is the case if it was written to by sequencer relayer).
+ASTRIA_SEQUENCER_RELAYER_PRESUBMIT_PATH=/path/to/presubmit.json
+
+# The path to which relayer will write its state after submitting to Celestia.
+# A file must exist at this path and the file must be readable and writable.
+# The file must either be empty or contain a JSON object that can be read by
+# by Relayer (this is the case if it was written to by sequencer relayer).
+ASTRIA_SEQUENCER_RELAYER_POSTSUBMIT_PATH=/path/to/postsubmit.json
```

Comment:
what's the reasoning for having two different files for the states? why not one file of the current state?

Reply:
This is a common pattern - if you use one file and you get a corruption (service crashes/gets killed while writing, etc) you lose the entire file. By separating this you will retain the old state (if writing `presubmit` failed) or retain information about what you tried to submit.

Thinking about it, `presubmit` should actually also store the last submitted state, because losing that is arguably worse.